terraform/a2ha-terraform/modules/aws_output/main.tf

```diff
@@ -11,6 +11,16 @@ locals {
       automate_fqdn             = var.automate_fqdn
       automate_frontend_urls    = var.automate_frontend_urls
     })
+
+    copy_files = [
```

Comment:
`copy_files` (vars in general) can be named better and should describe domain/context